Resources/config/orm.xml

```diff
@@ -144,6 +144,10 @@
             <argument type="tagged" tag="security.remember_me_handler" />
             <tag name="doctrine.event_subscriber" />
         </service>
+        <service id="doctrine.orm.listeners.pdo_session_handler_schema_subscriber" class="Symfony\Bridge\Doctrine\SchemaListener\PdoSessionHandlerSchemaSubscriber">
+            <argument type="service" id="Symfony\Component\HttpFoundation\Session\Storage\Handler\PdoSessionHandler" on-invalid="null" />
```

Comment:
Instead of harcoding the name of the pdo handler service, I think we'd better reference `session.handler` here.
This would go together with a change on PdoSessionHandlerSchemaSubscriber I'm going to post on https://github.com/symfony/symfony/pull/48059

Reply:
ok!